### PR TITLE
Bootstrap integration (Staff Dashboard & Application Template show)

### PR DESCRIPTION
**Pull request by enragednuke**
As per request by @dfaulken, this is now pushing to a separate `refactor` branch like `ft-refactor` on `round-three`.

See the old PR for visuals #225

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'bundler', '>= 1.8.4'
```
Comment (dfaulken) [inline Gemfile]: Does it work for bundler to bundle bundler?

File: app/assets/stylesheets/application_drafts/edit.scss [hunk 2]
```diff
@@ -23,13 +23,14 @@ html {
...
   display: flex;
+  padding-bottom: 20px;
   .form_column{
     flex:3;
     //list-style-type:none;
     .header_row{
       padding: .5em;
       display: flex;
-      height: 2em;
+      //height: 2em;
```
Comment (dfaulken) [inline app/assets/stylesheets/application_drafts/edit.scss]: Better to just remove

**Comment by dfaulken** [comment]
Looks good; thanks for bearing with me on this process.

Other than the bundler question this looks good.

File: Gemfile (hunk 1 above)
Comment (werebus) [inline Gemfile]: Actually, I don't see why it wouldn't.  But it's also probably unnecessary.

Comment (enragednuke) [inline Gemfile]: I likely put it there for *some* reason in the original PR, but it probably doesn't matter now. I'll see when I get in.

File: app/assets/stylesheets/application_drafts/edit.scss (hunk 2 above)
Comment (enragednuke) [inline app/assets/stylesheets/application_drafts/edit.scss]: I agree, I think I was messing with various values and accidentally left it in. Must've missed this when I was scanning for comments.

**Comment by enragednuke** [comment]
@dfaulken The bundler gem wasn't needed (I'm still trying to think of why it was there in the first place). As long as checks pass it should be good to go.

**Pull request merged**